cmd/supervisor/supervisor.go

```diff
@@ -19,43 +19,66 @@ package main
 
 import (
 	"flag"
-	"log"
 	"os"
 	"path/filepath"
+	"runtime"
+
+	"github.com/rs/zerolog/log"
 
 	"github.com/mysteriumnetwork/node/supervisor/config"
 	"github.com/mysteriumnetwork/node/supervisor/daemon"
 	"github.com/mysteriumnetwork/node/supervisor/install"
+	"github.com/mysteriumnetwork/node/supervisor/logconfig"
 )
 
 var (
 	flagInstall = flag.Bool("install", false, "Install or repair myst supervisor")
+	logFilePath = flag.String("log-path", "", "Supervisor log file path")
 )
 
 func main() {
 	flag.Parse()
 
 	if *flagInstall {
-		log.Println("Installing supervisor")
+		log.Info().Msg("Installing supervisor")
 		path, err := thisPath()
 		if err != nil {
-			log.Fatalln("Failed to determine supervisor's path:", err)
+			log.Fatal().Err(err).Msg("Failed to determine supervisor's path")
 		}
 		err = install.Install(install.Options{
 			SupervisorPath: path,
 		})
 		if err != nil {
-			log.Fatalln("Failed to install supervisor:", err)
+			log.Fatal().Err(err).Msg("Failed to install supervisor")
 		}
+		log.Info().Msg("Supervisor installed")
 	} else {
-		log.Println("Running myst supervisor daemon")
+		logPath := *logFilePath
+		if logPath == "" {
+			logPath = defaultLogPath()
+		}
+
+		if err := logconfig.Configure(logPath); err != nil {
+			log.Fatal().Err(err).Msg("Failed to configure logging")
+		}
+
+		log.Info().Msg("Running myst supervisor daemon")
 		supervisor := daemon.New(&config.Config{})
 		if err := supervisor.Start(); err != nil {
-			log.Fatalln("Error running supervisor:", err)
+			log.Fatal().Err(err).Msg("Error running supervisor")
 		}
 	}
 }
 
+func defaultLogPath() string {
+	if runtime.GOOS == "darwin" || runtime.GOOS == "linux" {
+		return "/var/log/myst_supervisor"
+	}
+
+	// On Windows default log file location will be under C:\Windows\system32\myst_supervisor.log
+	return "./myst_supervisor"
```

Comment:
Never write anything to system dirs. It would be the best if we could use eventlog on windows, and not bother with file rotation/location at all. WDYT

Reply:
We already logging to file at `~/.mysterium.log`. What about having this one next to it `~/.supervisor.log`?

Reply:
Changed default log path to `C:\Windows\system32\config\systemprofile\AppData\Local\MystSupervisor\myst_supervisor.log`